test/e2e/scheduling/preemption.go

```diff
@@ -98,10 +98,13 @@ var _ = SIGDescribe("SchedulerPreemption [Serial]", func() {
 		framework.ExpectNoError(err)
 	})
 
-	// This test verifies that when a higher priority pod is created and no node with
-	// enough resources is found, scheduler preempts a lower priority pod to schedule
-	// the high priority pod.
-	ginkgo.It("validates basic preemption works", func() {
+	/*
```

Comment:
nit: Would you mind correcting the typo in L125: `let not's` -> `let's not`.

Reply:
done